src/coreclr/jit/emitarm64.cpp

```diff
@@ -9836,7 +9836,7 @@ void emitter::emitIns_R_S(instruction ins, emitAttr attr, regNumber reg1, int va
             fmt      = IF_SVE_IE_2A;
 
             // TODO-SVE: Don't assume 128bit vectors
-            scale        = NaturalScale_helper(EA_16BYTE);
+            scale        = 4;
```

Comment:
wondering why is this change? It will return `4` here, right?

Reply:
>wondering why is this change? It will return 4 here, right?

Agreed. The changes for the mask load/stores were the required ones. 

Reverted.